p2p/Cargo.toml

```diff
@@ -19,6 +19,7 @@ arbitrary = ["dep:arbitrary", "bitcoin/arbitrary"]
 
 [dependencies]
 bitcoin = { path = "../bitcoin/", default-features = false }
+bitcoin-primitives = { path = "../primitives", version = "1.0.0-rc.0", default-features = false }
```

Comment:
Should we just call this `primitives` following the style of `units` below? Or vice versa?

Reply:
Yeah, good idea.